src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -2206,6 +2219,63 @@ private List<ModificationInfos> execLineAttachToVoltageLevel(NetworkStoreListene
         return inspectable;
     }
 
+    private List<ModificationInfos> execLinesAttachToSplitLines(NetworkStoreListener listener,
+                                                              LinesAttachToSplitLinesInfos linesAttachToSplitLinesInfos,
+                                                              UUID reportUuid) {
+        Network network = listener.getNetwork();
+
+        ReporterModel reporter = new ReporterModel(NETWORK_MODIFICATION_REPORT_KEY, NETWORK_MODIFICATION_REPORT_NAME);
+        Reporter subReporter = reporter.createSubReporter("linesAttachToSplitLines", "Lines attach to split lines");
+
+        List<ModificationInfos> inspectable = doAction(listener, () -> {
+            if (listener.isApplyModifications()) {
+                Line line1 = network.getLine(linesAttachToSplitLinesInfos.getLineToAttachTo1Id());
+                Line line2 = network.getLine(linesAttachToSplitLinesInfos.getLineToAttachTo2Id());
+                Line line = network.getLine(linesAttachToSplitLinesInfos.getAttachedLineId());
+                if (line1 == null) {
+                    throw new NetworkModificationException(LINE_NOT_FOUND, linesAttachToSplitLinesInfos.getLineToAttachTo1Id());
+                }
+                if (line2 == null) {
+                    throw new NetworkModificationException(LINE_NOT_FOUND, linesAttachToSplitLinesInfos.getLineToAttachTo2Id());
+                }
+
+                if (line == null) {
+                    throw new NetworkModificationException(LINE_NOT_FOUND, linesAttachToSplitLinesInfos.getAttachedLineId());
+                }
```

Comment:
these checks to test the lines existence are already made in ReplaceTeePointByVoltageLevelOnLine apply method

Reply:
fixed